crates/scmi/src/main.rs

```diff
@@ -0,0 +1,182 @@
+// SPDX-FileCopyrightText: Red Hat, Inc.
+// SPDX-License-Identifier: Apache-2.0
+// Based on implementation of other devices here, Copyright by Linaro Ltd.
+
+//! vhost-user daemon implementation for
+//! [System Control and Management Interface](https://developer.arm.com/Architectures/System%20Control%20and%20Management%20Interface)
+//! (SCMI).
+//!
+//! Currently, the mandatory parts of the following SCMI protocols are implemented:
+//!
+//! - base
+//! - sensor management
+//!
+//! As for sensor management, support for industrial I/O (IIO) Linux devices
+//! and a fake sensor device is implemented.
+//!
+//! The daemon listens on a socket that is specified using `--socket-path`
+//! command line option.  Usually at least one exposed device is specified,
+//! which is done using `--device` command line option.  It can be used more
+//! than once, for different devices.  `--device help` lists the available
```

Comment:
I tried:

```
$ ./target/debug/vhost-user-scmi --device help
error: the following required arguments were not provided:
  --socket-path <SOCKET_PATH>

Usage: vhost-user-scmi --socket-path <SOCKET_PATH> --device <DEVICE>...

For more information, try '--help'.
```

Then adding the socket path it worked, but initially I thought was not working.
What about adding the list of device supported in the standard `--help` page?

Reply:
I don't like too verbose --help (it wouldn't be that verbose now but may become in future), so I added --help-devices for the purpose and removed `-d help`. The complaint about missing --socket-path is resolved too. I believe help is well behaved now.